docker-compose/monitor/docker-compose-v2.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.5"
+services:
+  jaeger:
+    networks:
+      - backend
+    # TODO: Need to use the local image here while making changes 
+    image: jaegertracing/jaeger
```

Comment:
follow the same pattern as in the other compose file - env var defaulted to `latest` but `make dev` uses local build